docs/topics/jvm-api-guidelines-readability.md

```diff
@@ -119,20 +119,20 @@ fun a(block: A.() -> Unit): A
 
 ## Use constructor-like functions where applicable
 
-Sometimes, it makes sense to simplify your API's appearance by using constructor-like functions. A constructor-like function 
-is a function whose name starts with a capital letter so it looks like a constructor. This approach can make your library 
+Sometimes, you can simplify your API's appearance by using constructor-like functions. A constructor-like function 
+is a function whose name starts with a capital letter, so it looks like a constructor. This approach can make your library 
 easier to understand.
 
-For example, you want to introduce some [Option type](https://en.wikipedia.org/wiki/Option_type) in your library:
+For example, you want to introduce an [Option type](https://en.wikipedia.org/wiki/Option_type) in your library:
 
 ```kotlin
 sealed interface Option<T>
 class Some<T : Any>(val t: T) : Option<T>
 object None : Option<Nothing>
 ```
 
-You can define implementations of all the `Option` interface methods — `map()`, `flatMap()`, and so on. But each time 
-your API users create such an `Option`, they must write some logic and check what they create. For example:
+You can define implementations of all the `Option` interface methods &mdash; `map()`, `flatMap()`, and so on. But each time 
```

Comment:
```suggestion
You can define implementations of all the `Option` interface methods – `map()`, `flatMap()`, and so on. But each time 
```

Reply:
We use en dash everywhere instead of em dash https://docs.google.com/document/d/1mUuxK4xwzs3jtDGoJ5_zwYLaSEl13g_SuhODdFuh2Dc/edit#heading=h.32lwfrbt0ymf
We can change it one day, but for now we would like our docs to be consistent.

Reply:
Ok. But just FYI, that's not what en dash is for.